src/shared/utilities/vsCodeUtils.ts

```diff
@@ -114,3 +115,8 @@ export function getChannelLogger(channel: vscode.OutputChannel): ChannelLogger {
             })
     })
 }
+
+export async function getStartPort(): Promise<number> {
```

Comment:
Could we move this to a debug utils file? This file (vscodeutils.ts) doesn't strongly articulate what it is responsible for, and any decluttering we can do is a win.

Reply:
I'll deem that out of scope for this change since this function was essentially already here. Do you think it'd be worth it to create an issue to do away with this file entirely? IDK what 90% of this stuff is....